tests/test_build.py

```diff
@@ -78,12 +78,12 @@ def test_build_all():
 
 
 @with_tempdir
-def test_master_doc_not_found(tmpdir):
-    (tmpdir / 'conf.py').write_text('master_doc = "index"')
-    assert tmpdir.listdir() == ['conf.py']
+def test_master_doc_not_found(tempdir):
```

Comment:
now `tmpdir` is naked pytest fixture, otherwise `tempdir` is temporary directory object that is wrapped with `path`